app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -135,6 +138,35 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
     } catch (error) {
       console.error('Could not load week data:', error);
```

Comment:
This should be throw Error since the stack track will tell us where it is from.

Reply:
✅ fixed!